.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+Thanks for your interest in the project. I appreciate bugs filed and PRs submitted!
```

Comment:
If we keep this section, then we should update to 'we' and make some other minor changes:

Thanks for your interest in the OpenCue project. We appreciate bug reports and feature requests.

To help the project address your issue, fill out this template with the requested information.

If your issue isn't related to the documentation, file an issue in the main OpenCue repository:

https://github.com/AcademySoftwareFoundation/OpenCue/issues/new/choose

Reply:
That's great and also looks good, I will update the changes.